src/EventStore.Core/Services/Transport/Grpc/ServerFeatures.cs

```diff
@@ -26,19 +29,22 @@ internal partial class ServerFeatures
 						MethodName = x.Method.Name.ToLower(),
 						ServiceName = x.Method.ServiceName.ToLower(),
 					};
+
 					if (x.Method.ServiceName.Contains("PersistentSubscriptions")) {
 						method.Features.AddRange(new[] {"stream", "all"});
 					} else if (x.Method.ServiceName.Contains("Streams") && x.Method.Name.Contains("Read")) {
 						method.Features.AddRange(new[] {"position", "events"});
+					} else if (x.Method.ServiceName.Contains("Streams") && x.Method.Name.Contains("BatchAppend")) {
+						method.Features.Add("deadlineduration");
```

Comment:
should this have some kind of separation between the words (casing, or hyphen), or have we already got other ones where the words are just concatenated

Reply:
I think underscore is the way to go here